docs/source/datasets/index.rst

```diff
@@ -117,6 +117,9 @@ Bureau field representatives across the country through both personal and teleph
 work activity and income, veteran status, school enrollment, contingent employment, worker displacement, job tenure, and more. To find out more
 about CPS, please visit the `CPS homepage <https://www.census.gov/programs-surveys/cps.html>`_. 
 
+To find out more about how to generate a simulation of ACS using Pseudopeople, see :func:`pseudopeople.interface.generate_american_community_survey`. 
```

Comment:
I think the API links for the two surveys should be split out onto separate lines so that the placement of the links (and the wording) is consistent across all datasets.